src/AutoMapper/Configuration/IProfileConfiguration.cs

```diff
@@ -12,6 +12,8 @@ namespace AutoMapper.Configuration
     public interface IProfileConfiguration
     {
         IEnumerable<IMemberConfiguration> MemberConfigurations { get; }
+        bool? FieldMappingEnabled { get; }
+        bool? MethodMappingEnabled { get; }
```

Comment:
Another approach would be to use `ShouldMap*` instead. Make them expressions and check for `false`.